target/bin/setup

```diff
@@ -13,89 +11,80 @@ function _usage
   printf '%s' "${PURPLE}SETUP${RED}(${YELLOW}1${RED})
 
 ${ORANGE}NAME${RESET}
-    ${SCRIPT:-${0}} - 'docker-mailserver' Administration & Configuration script
+    setup - 'docker-mailserver' Administration & Configuration CLI
 
 ${ORANGE}SYNOPSIS${RESET}
-    ./${SCRIPT:-${0}} [ OPTIONS${RED}...${RESET} ] COMMAND [ help ${RED}|${RESET} ARGUMENTS${RED}...${RESET} ]
+    setup [ OPTIONS${RED}...${RESET} ] COMMAND [ help ${RED}|${RESET} ARGUMENTS${RED}...${RESET} ]
 
     COMMAND ${RED}:=${RESET} { email ${RED}|${RESET} alias ${RED}|${RESET} quota ${RED}|${RESET} dovecot-master ${RED}|${RESET} config ${RED}|${RESET} relay ${RED}|${RESET} debug } SUBCOMMAND
 
 ${ORANGE}DESCRIPTION${RESET}
-    This is the main administration script that you use for all your interactions with
```

Comment:
If you want to keep some of this description that was removed, you would want to relocate it to `setup.sh` in some manner, or the docs on `setup.sh`. I don't think it's relevant anymore to be interleaved into the main `setup` command help output.

---

I was not able to provide a web UI suggestion for changes and edited the file myself. The commit message detais:

- Remove dynamic support for showing command invoker via `${0}` for `./setup.sh` or `/usr/local/bin/setup`, just show `setup` command.
- Remove bulk of description text that isn't applicable to the `setup` command, but `./setup.sh` only.
- Revised description for help usage, as attempting `setup email help` or similar is not valid. Clarify to user they must be more specific.
- Revised the 2nd example description.
- Changed references of "script" to "CLI" which is now more accurate representation of the `setup` command.

Reply:
Note that most commands call scripts that will still output `./setup.sh` in their `help` output.